### PR TITLE
Add price to person

### LINKED ISSUE #84: Add missing price functionality

### PR DESCRIPTION
**Pull request by ZechariahTan**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DriveCommand.java [hunk 1]
```diff
@@ -44,6 +46,10 @@
     public static final String MESSAGE_NO_COMMUTERS = "No commuters were selected.";
     public static final String MESSAGE_DRIVE_SUCCESS = "Assigned %s to: %s";
 
+
+    //todo remove STUD_VALID_PRICE declaration
+    private static final Price STUD_VALID_PRICE = new Price("1.69");
```
Comment (chewterence) [inline src/main/java/seedu/address/logic/commands/DriveCommand.java]: did you mean stub?

File: src/main/java/seedu/address/logic/commands/DriveCommand.java [hunk 2]
```diff
@@ -103,7 +109,10 @@ private static Passenger assignDriverToPassenger(Passenger passengerToEdit, Driv
         TripDay updatedTripDay = passengerToEdit.getTripDay();
         TripTime updatedTripTime = passengerToEdit.getTripTime();
 
-        return new Passenger(updatedName, updatedPhone, updatedAddress, updatedTripDay, updatedTripTime, driver,
+        //todo remove price stud usage
+        Price price = STUD_VALID_PRICE;
```
Comment (chewterence) [inline src/main/java/seedu/address/logic/commands/DriveCommand.java]: did you mean stub?

Comment (chewterence) [reply src/main/java/seedu/address/logic/commands/DriveCommand.java]: i noticed the same potential problem in the rest of the code

File: src/main/java/seedu/address/model/person/passenger/Passenger.java [hunk 3]
```diff
@@ -25,18 +25,21 @@
     private final Address address;
     private final TripDay tripDay;
     private final TripTime tripTime;
+    private final Price price;
```
Comment (JoelHo) [inline src/main/java/seedu/address/model/person/passenger/Passenger.java]: Should price be Optional<Price>, similar to driver?

File: src/test/java/seedu/address/testutil/PassengerUtil.java [hunk 4]
```diff
@@ -12,13 +13,17 @@
...
+import seedu.address.model.person.passenger.Price;
 import seedu.address.model.tag.Tag;
 
 /**
  * A utility class for Passenger.
  */
 public class PassengerUtil {
 
+    //todo remove STUD declaration
+    private static final Price STUD_VALID_PRICE = new Price("1.69");
```
Comment (JoelHo) [inline src/test/java/seedu/address/testutil/PassengerUtil.java]: STUD->STUB

**Review by JoelHo — CHANGES_REQUESTED** [review]

**Review by JoelHo — APPROVED** [review]

**Pull request merged**